H/HiGHS/highs_common.jl

```diff
@@ -44,33 +49,9 @@ else
     fi
 fi
 make install
+if [[ "${BUILD_SHARED}" == "OFF"]]
+    # Delete the static library to save space
+    rm ${destdir}/lib/libhighs*
```

Comment:
You'd just have an empty directory as far as I can see:
```suggestion
    rm -r ${prefix}/lib
```